lib/rubocop/cop/style/line_end_concatenation.rb

```diff
@@ -54,8 +54,11 @@ def on_new_investigation
 
         private
 
+        # rubocop:disable Metrics/AbcSize
         def check_token_set(index)
-          predecessor, operator, successor = processed_source.tokens[index, 3]
+          predecessor = processed_source.tokens[index]
```

Comment:
Is a such a change really making a difference? I'm curious what's the reasoning behind it.

Reply:
(I'm assuming it's to avoid allocating the 3-elemenent array that we were deconstructing)

Reply:
Btw, this also looks like a good idea for a style cop - preferring array deconstruction unless you know what you're doing (optimizing allocations). 😆 Might be a good idea in this particular case to assign the tokens to some local to avoid a bit of repetition.

Reply:
It was `234_337` objects allocated (10mb), and then completely disappeared from the memory report.